src/schema/rules/dataset_metadata.yaml

```diff
@@ -0,0 +1,53 @@
+dataset_description:
+    selectors:
+    - path == /dataset_description.json
+    fields:
+        Name: REQUIRED
+        BIDSVersion: REQUIRED
+        DatasetType: RECOMMENDED
+        License: RECOMMENDED
+        Authors: OPTIONAL
+        Acknowledgements: OPTIONAL
+        HowToAcknowledge: OPTIONAL
+        Funding: OPTIONAL
+        EthicsApprovals: OPTIONAL
+        ReferencesAndLinks: OPTIONAL
+        DatasetDOI: OPTIONAL
+        GeneratedBy: RECOMMENDED
+        GeneratedBy[].Name: REQUIRED
+        GeneratedBy[].Version: RECOMMENDED
+        GeneratedBy[].Description: OPTIONAL
+        GeneratedBy[].CodeURL: OPTIONAL
+        GeneratedBy[].Container: OPTIONAL
+        GeneratedBy[].Container.Type: RECOMMENDED
+        GeneratedBy[].Container.Tag: RECOMMENDED
+        GeneratedBy[].Container.URI: RECOMMENDED
+        SourceDatasets: RECOMMENDED
+
+derivative_description:
+    selectors:
+    - path == /dataset_description.json
+    - dataset.dataset_description.DatasetType == "derivative"
+    fields:
+        GeneratedBy: REQUIRED
+
+dataset_description_with_genetics:
+    selectors:
+    - path == /dataset_description.json
+    - dataset.files contains "/genetic_info.json"
```

Comment:
If we're searching for files via paths, would we want to include search patterns here? E.g., `dataset.files contains `/sub-{subject}/*/pet/*` to determine if the subject has PET data?

Reply:
I don't think that level of flexibility is called for at this point.